src/nft.vy

```diff
@@ -272,6 +277,7 @@ def __init__(name_: String[25], symbol_: String[5], base_uri_: String[80], name_
     self._transfer_ownership(msg.sender)
     self.is_minter[msg.sender] = True
     log RoleMinterChanged(msg.sender, True)
```

Comment:
Create an internal function that changes minter and log event. Use it in relevant function such as this and set_minter

Reply:
ack: implemented